IdentityServer/v6/BFF/SplitHosts/BackendHost/Startup.cs

```diff
@@ -0,0 +1,100 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using Duende.Bff.Yarp;
+using Duende.Bff;
+
+namespace BackendHost
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddCors(opt => 
+            {
+                opt.AddDefaultPolicy(policy =>
+                {
+                    policy.WithOrigins("https://localhost:5011")
+                        .AllowAnyHeader()
+                        .AllowAnyMethod()
```

Comment:
This is very permissive CORS policy. Would a more locked down set of policies be helpful or just distracting?

Reply:
yes - it needs to explicitly allow only the required headers + the X-CSRF header.

Reply:
Done!